docs/ui/index.rst

```diff
@@ -0,0 +1,167 @@
+          is shown as the user puts together a query to see the name property
+          for a user-defined type called Book. A filter on the object's id
+          and a limit to the number of object types returned are being set.
+          The Editor icon is a blue square resembling a pad, with an orange
+          line resembling a pencil on top.
+    :width: 100%
+
+The Editor page's *query editor* is recommended for:
+
+- Users experimenting with various raw queries who want quick visual
+  point-and-click access to past queries in order to call them up again
+  and refine them.
+
+.. image:: images/query_editor.png
+    :alt: The editor tab in the EdgeDB UI editor page, showing a query
+          appended with the analyze keyword to analyze performance. The
+          performance results show up in a graph on the right, with separate
+          colored rectangles for each link traversed by the query.
+    :width: 100%
+
+The UI's REPL is recommended for:
+
+- Users comfortable with EdgeQL.
+
+.. image:: images/ui_repl.png
+    :alt: The REPL page in the EdgeDB UI. The icon is a blue right-facing
+          angle bracket followed by an orange underscore, representing a
+          cursor awaiting user input. The output for a query on a
+          user-defined type called Person is displayed.
+    :width: 100%
+
+Additionally, users who spend a lot of time comparing raw queries may
+wish to give the CLI's REPL a try. A general rule of thumb is that the
+UI's REPL provides a slicker experience and more verbose output, while
+the CLI's REPL is a more performant tool that usually returns query results
+instantaneously.
+
+For example, the output from selecting objects of a type in the UI's REPL will
+show more information on a scalar object's type name:
+
+.. code-block:: edgeql-repl
+    :caption: CLI REPL
+
+    db> select Sailor {*};
+    default::Sailor {
+      id: f0b4aaf0-be4c-11ee-b84b-6b87ec260333,
+      cents: 0,
+      dollars: 0,
+      pence: 149,
+      pounds: 14,
+      shillings: 28,
+      total_cents: 0,
+      total_pence: 4069,
+      approx_wealth_in_pounds: 17,
+    },
+
+.. code-block:: edgeql-repl
+    :caption: UI REPL
+
+    db> select Sailor {*};
+    default::Sailor {
+      id: <uuid>'f0b4aaf0-be4c-11ee-b84b-6b87ec260333',
+      cents: <default::Money>0,
+      dollars: <default::Money>0,
+      pence: <default::Money>149,
+      pounds: <default::Money>14,
+      shillings: <default::Money>28,
+      total_cents: 0,
+      total_pence: 4069,
+      approx_wealth_in_pounds: 17,
+    },
+
+One more example of CLI vs. UI output, showing a user-defined function:
+
+.. code-block::
+
+  # CLI REPL output
+  'function default::get_url() ->  std::str using
+  (<std::str>\'https://geohack.toolforge.org/geohack.php?params=\');'}
+
+  # UI REPL output
+  function default::get_url() -> std::str {
+  volatility := 'Immutable';
+  using (<std::str>'https://geohack.toolforge.org/geohack.php?params=');
+  }
```

Comment:
Could you update this example to mirror the one above, with two distinct code blocks and showing the query?

Reply:
Sure!